src/components/Employees/index.tsx

```diff
@@ -0,0 +1,214 @@
+import styles from './Employees.module.css';
+import employees from '@components/Employees/employeesData';
+import EmployeeItem from '@components/EmployeeItem';
+import { IEmployeeItem } from '@components/EmployeeItem/types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+import { useSearchParams } from 'react-router-dom';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+function Employees() {
+    const { t } = useTranslation();
+    const [searchParams, setSearchParams] = useSearchParams({});
+    const [searchActiveButtons, setSearchActiveButtons] = useState<string[]>(
+        []
+    );
+
+    function searchButtonExists(searchParam: string): boolean {
+        return searchActiveButtons.some(
+            (searchBtn) => searchBtn === searchParam
+        );
+    }
+
+    function employeeExists(
+        searchParam: IEmployeeItem,
+        filteredEmployees: IEmployeeItem[]
+    ): boolean {
+        return filteredEmployees.some((searchBtn) => searchBtn === searchParam);
+    }
+
+    function addFilter(target: string, keyParam: string) {
+        searchParams.append(keyParam, target);
+        setSearchParams(searchParams);
+        if (!searchButtonExists(target)) {
+            setSearchActiveButtons([...searchActiveButtons, target]);
+        }
+    }
+
+    function findFilters(filterArray: string[], target: string) {
+        const foundFilters: string[] = [];
+        filterArray.forEach((filter) => {
+            if (filter !== target) {
+                foundFilters.push(filter);
+            }
+        });
+        return foundFilters;
+    }
```

Comment:
```suggestion
    function findFilters(filters: string[], target: string) {
        return filters.filter((filter) => filter !== target);
    }
```

Reply:
I always forget that i don't need to write some functions that already exists and makes life easier.